utils/env.ts

```diff
@@ -0,0 +1,64 @@
+import 'dotenv/config';
+
+const MAX_ACCOUNTS = 10;
+
+export function getNodeUrl(network: string): string {
+  if (network) {
+    const uri = process.env[`NODE_URI_${network.toUpperCase()}`];
+    if (uri && uri !== '') {
+      return uri;
+    }
+  }
+  console.warn(`No node uri for network ${network}`);
+  return '';
+}
+
+export function getMnemonic(network: string): string {
+  const mnemonic = process.env[`${network.toUpperCase()}_MNEMONIC`] as string;
+  if (!mnemonic) {
+    console.warn(`No mnemonic for network ${network}`);
+    return 'test test test test test test test test test test test junk';
+  }
+  return mnemonic;
+}
+
+export function getPrivateKeys(network: string): string[] {
+  const privateKeys = [];
+  for (let i = 1; i <= MAX_ACCOUNTS; i++) {
+    const privateKey = process.env[`${network.toUpperCase()}_${i}_PRIVATE_KEY`];
+    if (!!privateKey) privateKeys.push(privateKey);
+  }
+  if (privateKeys.length == 0) {
+    console.warn(`No private keys for network ${network}`);
+    privateKeys.push('0x0000000000000000000000000000000000000000000000000000000000000bad');
```

Comment:
I think that if we do not have a fallback, if we were to test a `fork` and the local environment does not have any fallback like this line, it would fail trying to get accounts.
Not entirely sure how hardhat reacts when `accounts = []`